examples/hubspot/index.js

```diff
@@ -0,0 +1,106 @@
+const express = require('express');
+const { Client } = require('@hubspot/api-client');
+const everyauth = require('../..');
```

Comment:
Never a big fan of this - let's keep it as @fusebit/everyauth-express and add a comment in to `npm link` as appropriate if you want to run the local version - which most people won't.

Reply:
I left a similar comment in the Salesforce one,  I wouldn't add the npm link comment just yet, people will follow this examples in their own express app.

Reply:
Pinning the sample code to the module in the repo rather than on npm is actually simpler for everybody I think. You pull a repo (which you have to do anyways to get the sample code in the first place) and you are guaranteed it is always consistent. I'd find it confusing if a sample was demonstrating running a different version of the code than the one I just pulled.

Reply:
If the instructions indicate to do an `npm install`, then it's already pulling down the necessary package. It's usually something I end up having to fix because it means the code is "wrong" for actual usecases, when I've run into people doing this in the past.

Optimizing for the "run the local code" situation is actually counter-indicated on most repositories, too, fwiw, since usually HEAD is not a published version (and may or may not work, even).

Reply:
Changed to fixed dep.